pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -102,7 +102,10 @@ angular: teacherDashboard
       //  'remove': {method:'DELETE'},
       //  'delete': {method:'DELETE'}
          update: {method:'POST', url: 'v2/sections/:id/update'},
+         // TODO: This isn't very restful
+         allStudents: {method:'GET', url:'v2/sections/:id/students', isArray: true},
          addStudents: {method:'POST', url:'/v2/sections/:id/students', isArray: true},
+         moveStudents: {method:'POST', url:'/sections/:new_section_code/transfers'},
```

Comment:
Ah, so the new section code isn't figured out until the user enters it, so I cannot pre-generate the URL in haml. I'll probably have to change the API endpoint to take in the new section code as a param so I can use the ```CDO.studio_url``` method. I'll fix that up.